crates/playground/tsconfig.bak

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Do we need to commit the `.bak` file for any reason?

Reply:
Nope :)